lib/classes/class-errors.php

```diff
@@ -171,9 +158,15 @@ public function admin_notices() {
         $warnings = apply_filters( 'ud:warnings:admin_notices', $this->warnings, $this->args );
         
         if( !empty( $errors ) || !empty( $messages ) || !empty( $warnings ) ) {
+          //enqueue dismiss js for ajax requests
```

Comment:
I think you need to add another condition after printing messages instead of the current one. We have conditions below when notices and/or warnings are not showed. So if we echo anything we are loading script, in other case - ignore. Does it make sense?